src/ORM/DataList.php

```diff
@@ -2061,4 +2061,14 @@ public function chunkedFetch(int $chunkSize = 1000): iterable
             $currentChunk++;
         }
     }
+
+    /**
+     * Prepopulate any extension caches with the current dataclass and IDs of records
+     */
+    public function prepopulateCaches(): void
+    {
+        $dataClass = $this->dataClass;
+        $ids = $this->column('ID');
+        $this->extend('updatePrepopulateCaches', $dataClass, $ids);
```

Comment:
```suggestion
        $this->extend('updatePrepopulateCaches', $ids);
```
The dataclass can be accessed from the extension with `$this->getOwner()->dataClass()`

Reply:
Updated